File: CH3-desicion-sructure-and-boolean-logic/EX6-book-club-points/source-code/bookPointClub.cpp
#include <iostream>
using namespace std;

int main()
{
    int bookPurchased;
    int point;

    cout << "Number of books purchased : ";
    cin >> bookPurchased;

    if (bookPurchased < 1)
    {
        point = 0;
    }
    else
    {
        if (bookPurchased == 1)
        {
            point = 15;
        }
        else
        {
            if (bookPurchased == 3)
            {
                point = 30;
            }
            else
            {
                point = 60;
            }
        }
    }
    cout << "Points awarded : " << point << endl;
}

File: CH3-desicion-sructure-and-boolean-logic/EX6-book-club-points/bookPointClub.js
const input = require("prompt-sync")();

let bookPurchased;
let point;

bookPurchased = input("Number of books purchased :");

if (bookPurchased < 1) {
  point = 0;
} else {
  if (bookPurchased == 1) {
    point = 5;
  } else {
    if (bookPurchased == 2) {
      point = 15;
    } else {
      if (bookPurchased == 3) {
        point = 30;
      } else {
        point = 60;
      }
    }
  }
}

console.log("points awarded : " + point);
